UnityProject/Assets/Scripts/Player/LayDown.cs

```diff
@@ -0,0 +1,98 @@
+﻿using HealthV2;
+using Mirror;
+using Player.Movement;
+using UnityEngine;
+
+namespace Player
+{
+	public class LayDown : NetworkBehaviour
+	{
+		[SerializeField] private Transform sprites;
+		[SerializeField] private LivingHealthMasterBase health;
+		[SerializeField] private Rotatable playerDirectional;
+		[SerializeField] private PlayerScript playerScript;
+		[SerializeField] private Util.NetworkedLeanTween networkedLean;
+		private readonly Quaternion layingDownRotation = Quaternion.Euler(0, 0, -90);
+		private readonly Quaternion standingUp = Quaternion.Euler(0, 0, 0);
+
+		[SyncVar(hook = nameof(OnLayDown))] public bool IsLayingDown = false;
+
+
+		private void Awake()
+		{
+			playerScript ??= GetComponent<PlayerScript>();
+			playerDirectional ??= GetComponent<Rotatable>();
+			health ??= GetComponent<LivingHealthMasterBase>();
+		}
+
+		private void OnEnable()
+		{
+			if (CustomNetworkManager.IsServer == false)
+			{
+				UpdateManager.Add(EnsureCorrectState, 5f);
+			}
+		}
+
+		private void OnDisable()
+		{
+			if (CustomNetworkManager.IsServer == false)
+			{
+				UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, EnsureCorrectState);
+			}
+		}
+
+		public void EnsureCorrectState()
+		{
+			if (IsLayingDown || health.IsDead) { LayingDownLogic(true); } else { UpLogic(true); }
+		}
+
+		public void OnLayDown(bool oldValue, bool newValue)
+		{
+			if (newValue || health.IsDead)
+			{
+				LayingDownLogic();
+			}
+			else
+			{
+				UpLogic();
+			}
+			HandleGetupAnimation(newValue == false);
+		}
+
+		private void LayingDownLogic(bool forceState = false)
+		{
+			if (forceState) sprites.localRotation = layingDownRotation;
+			foreach (SpriteRenderer spriteRenderer in GetComponentsInChildren<SpriteRenderer>())
+			{
+				spriteRenderer.sortingLayerName = "Bodies";
+			}
+			playerScript.PlayerSync.CurrentMovementType  = MovementType.Crawling;
+			if (CustomNetworkManager.IsServer) playerDirectional.LockDirectionTo(true, playerDirectional.CurrentDirection);
+			if (CustomNetworkManager.IsServer) playerScript.OnLayDown?.Invoke();
```

Comment:
codacy is right, you just checked the very same thing. Merge it to that if statement. Make a body for the if and wrap it with brackets.

Reply:
fixed